### DISCUSSION TITLE
5월 8일 백엔드 주간회의

### DISCUSSION THREAD

**Discussion by nuyh99** [woowacourse/prolog#1268] [backend]
* `MemberController`, `MemberReactionController` 도메인 분석 정리
* 분석 결과를 어떻게 멘토님에게 검증받을 것인지 결정하기
* 다음 도메인 분석 범위 정하기

**Comment by BGuga** [comment]
# 5/8 주간 회의록

## 지난 주 근로 진행 현황

### 테스트 코드 작성

오션, 라온 : merge 완료

베베, 호이 : 코드 리뷰 대기 중

스플릿, 져니 : merge 완료

홍실, 아마란스 : merge 완료

우르, 주디 : 5/8 저녁까지 merge 완료 예정

### 마지막 pr 을 보낸 페어는 작업 내용을 main -> develop 로 최신화 한다.

## 분석 결과를 어떻게 멘토에게 검증받을 것인지 결정하기

### 리뷰 테스트 코드 기반으로 도메인을 이해한 후 이해한 내용을 [문서](https://github.com/woowacourse/prolog/discussions/1275)로 정리한다.

1. 각 팀에게 분석할 도메인 범위를 할당한다.
2. 배정받은 도메인에 대한 테스트를 작성한다.
3. 테스트를 통해 이해한 내용을 해당 테스트를 담당한 페어가 문서화 한다.
4. 정리한 내용을 멘토에게 검증받는다.

### 근로 진행 계획

> 수요일 
 테스트 코드 작성 및 문서 작성

> 금요일
테스트 코드, 문서 작성 마무리
이해한 내용을 검증하기 위해 멘토님에게 리뷰 요청

>월요일 
테스트 코드 및 문서 병합

## 다음 주  도메인 분석 범위

분석 도메인 : studylog

팀 1  : 스플릿

1. `AbstractStudylogDocumentService`
2. `CommentService`
3. `FakeStudylogDocumentService`
5. `FilterService`
6. `PopularStudylogService`
7. `StudylogDocumentService`
8. `StudylogLikeService`
9. `StudylogScrapService`

팀 2 : 우르

1. `StudylogService`
2. `StudylogSessionService`
3. `StudylogTagService`
4. `TagService`
5. `ViewedStudyLogCookieGenerator`
